src/objects/BatchedMesh.js

```diff
@@ -651,7 +651,7 @@ class BatchedMesh extends Mesh {
 		const drawRanges = this._drawRanges;
 		if ( geometryId >= drawRanges.length || drawRanges[ geometryId ].active === false ) {
 
-			return this;
+			throw new Error( `BatchedMesh: Invalid geometryId ${geometryId}. Geometry is either out of range or inactive.` );
```

Comment:
We should say that the geometry is "deleted" rather than "inactive". The "active" state isn't a concept exposed to end users.

Reply:
done !